xmake/rules/platform/linux/driver/driver_modules.lua

```diff
@@ -283,9 +283,9 @@ function link(target, opt)
         -- generate target.mod.c
         local orderfile = path.join(path.directory(targetfile_o), "modules.order")
         local symversfile = path.join(path.directory(targetfile_o), "Module.symvers")
-        argv = {"-m", "-a", "-o", symversfile, "-e", "-N", "-w", "-T", "-"}
+        argv = {"-m", "-a", "-o", symversfile, "-e", "-N", "-w", "-T", orderfile}
         io.writefile(orderfile, targetfile_o .. "\n")
-        os.vrunv(modpost, argv, {stdin = orderfile})
+        os.vrunv(modpost, argv)
```

Comment:
有没有测过 兼容低版本么？

Reply:
ubuntu 22.04 源里有 5.15.x 5.17.x 5.19.x，每个版本选取了一个测试

比如 linux-headers-5.15.0-88-generic 的 Makefile.modpost 脚本就是用的 `-T -` 的方式，并且测试了 `-T orderfile` 也能通过编译
```Makefile
# /usr/src/linux-headers-5.15.0-86-generic/scripts/Makefile.modpost
...
# Read out modules.order to pass in modpost.
# Otherwise, allmodconfig would fail with "Argument list too long".
quiet_cmd_modpost = MODPOST $@
      cmd_modpost = sed 's/\.ko$$/$(mod-prelink-ext)\.o/' $< | $(MODPOST) -T -
...
```

此外还测试了 linux-headers-5.17.0-1003-oem、linux-headers-5.19.0-50-generic 都可以通过

Reply:
ok, thanks